include/enums.h

```diff
@@ -4832,7 +4841,7 @@ enum EffectInstanceFlags {
     EFFECT_INSTANCE_FLAGS_2                 = 0x00000002,
     EFFECT_INSTANCE_FLAGS_4                 = 0x00000004,
     EFFECT_INSTANCE_FLAGS_8                 = 0x00000008,
-    EFFECT_INSTANCE_FLAGS_10                = 0x00000010,
+    EFFECT_INSTANCE_FLAGS_10                = 0x00000010, // mark ready for cleanup
```

Comment:
worth naming the actual flag?

Reply:
I suppose so. I was going to wait for the function managing their lifecycle to be decomped.

Reply:
fine with me! :)